core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,18 +1,183 @@
-<div class="modal-header">
-  <p>
-    Are you sure you want to unpublish this story? Unpublishing a story will
-    delete any corresponding translation opportunities from the Contributor
-    Dashboard and reject any related pending translation suggestions.
-  </p>
+<div *ngIf="platformFeatureService.status.SerialChapterLaunchCurriculumAdminView.isEnabled">
+  <div class="modal-header">
+    <p>
+      You're attempting to unpublish chapters, please specify a reason and note the warning
+      message before proceeding.
+    </p>
+  </div>
+  <div class="modal-body">
+    <div class="reason-select">
+      <span>Select Reason:</span>
+
+      <div ngbDropdown class="dropdown">
+        <div ngbDropdownToggle
+             aria-label="Dropdown toggle to select unpublishing reason"
+             aria-haspopup="menu"
+             class="selected-reason-text">
+          {{ selectedReasonText }}
+        </div>
+        <div ngbDropdownMenu
+             class="dropdown-menu">
```

Comment:
Ditto as above, it is difficult to  notice that `dropdown` and `dropdown-menu` are two different classes and are not applied to all dropdown menus and are specific to one dropdown menu only

Reply:
@Rijuta-s I have changed the class names. But they are kept similar to the corresponding angular dropdown directives.